src/auto-complete.jsx

```diff
@@ -64,7 +64,13 @@ const AutoComplete = React.createClass({
       open: this.props.open,
     };
   },
-
+  componentWillReceiveProps: function(nextProps) {
+    if (this.state.searchText !== nextProps.searchText) {
```

Comment:
`if (this.props.searchText !== nextProps.searchText) {`

Would probably be better